build/bin.NOTICE

```diff
@@ -0,0 +1,162 @@
+  Apache Log4j
+  Copyright 1999-2019 Apache Software Foundation
+
+  This product includes software developed at
+  The Apache Software Foundation (http://www.apache.org/).
+
+  ResolverUtil.java
+  Copyright 2005-2006 Tim Fennell
+
+  Dumbster SMTP test server
+  Copyright 2004 Jason Paul Kitchen
+
+  TypeUtil.java
+  Copyright 2002-2012 Ramnivas Laddad, Juergen Hoeller, Chris Beams
+
+  picocli (http://picocli.info)
+  Copyright 2017 Remko Popma
+
+Apache Log4j Scala API (lib/org.apache.logging.log4j.log4j-api-scala_<VERSION>.jar)
+  Copyright 2016-2018 Apache Software Foundation
+
+  This product includes software developed at
+  The Apache Software Foundation (http://www.apache.org/).
+
+  AsciidocPlugin.scala
+  Copyright (c) 2008, 2009, 2010, 2011 Josh Suereth, Steven Blundy, Josh Cough,
+  Mark Harrah, Stuart Roebuck, Tony Sloane, Vesa Vilhonen, Jason Zaugg
+
+Apache Xerces Java (lib/xerces.xercesImpl-<VERSION>.jar)
+  Apache Xerces Java
+  Copyright 1999-2020 The Apache Software Foundation
+
+  This product includes software developed at
+  The Apache Software Foundation (http://www.apache.org/).
+
+  Portions of this software were originally based on the following:
+    - software copyright (c) 1999, IBM Corporation., http://www.ibm.com.
+    - software copyright (c) 1999, Sun Microsystems., http://www.sun.com.
+    - voluntary contributions made by Paul Eng on behalf of the 
+      Apache Software Foundation that were originally developed at iClick, Inc.,
+      software copyright (c) 1999.
+
+Apache XML Commons Resolver (lib/xml-resolver.xml-resolver-<VERSION>.jar)
+  Apache XML Commons Resolver
+  Copyright 2006 The Apache Software Foundation.
+
+  This product includes software developed at
+  The Apache Software Foundation http://www.apache.org/
+
+  Portions of this code are derived from classes placed in the
+  public domain by Arbortext on 10 Apr 2000. See:
+  http://www.arbortext.com/customer_support/updates_and_technical_notes/catalogs/docs/README.htm
+
+Apache XML Commons XML APIs (lib/xml-apis.xml-apis-<VERSION>.jar)
+  Apache XML Commons XML APIs
+  Copyright 1999-2009 The Apache Software Foundation.
+
+  This product includes software developed at
+  The Apache Software Foundation (http://www.apache.org/).
+
+  Portions of this software were originally based on the following:
+    - software copyright (c) 1999, IBM Corporation., http://www.ibm.com.
+    - software copyright (c) 1999, Sun Microsystems., http://www.sun.com.
+    - software copyright (c) 2000 World Wide Web Consortium, http://www.w3.org
+
+JDOM2 (lib/org.jdom.jdom2-<VERSION>.jar)
+  Copyright (C) 2000-2012 Jason Hunter & Brett McLaughlin.
+
+  All rights reserved.
+
+  This product includes software developed by the
+  JDOM Project (http://www.jdom.org/).
+
+Jansi (lib/org.fusesource.jansi.jansi-<VERSION>.jar)
+  Copyright (C) 2009, Progress Software Corporation and/or its
+  subsidiaries or affiliates.  All rights reserved.

```

Comment:
We might not distribute Jansi? I think that's only needed by daffodil-cli, so depending on whether or not we include daffodil-cli this dependency might not be pulled in.

Reply:
:+1:  I'll make a pass and prune the CLI dependencies.